src/server/Server.js

```diff
@@ -24,7 +25,16 @@ const apiAuthentication = process.env.AUTHENTICATION
   ? {enabled: true, users: authenticationUsers(process.env.AUTHENTICATION) }
   : {enabled: false};
 
+
+
+const enableSsl = process.env.CERT_DOMAINNAMES ? true : false;
+
 const app = setupExpressApp(mailHandler, apiAuthentication, process.env.API_KEY);
 
-const serverPort = 3000;
-app.listen(serverPort, () => logger.info(`Started service on port ${serverPort}!`));
+if(enableSsl) {
+  autoSsl(app);
+} else {
+  const serverPort = 3000;
+  app.listen(serverPort, () => logger.info(`Started service on port ${serverPort}!`));
+}
```

Comment:
Can we log out in which mode we are starting the express application?

Reply:
I can only do a simple log statement before starting `autoSsl(app)` as I'm afraid we don't have the callback like we do when just starting the express app.

Is that ok for you?

Reply:
Sure, thanks :)